turbo.json

```diff
@@ -37,6 +37,12 @@
     "dev": {
       "cache": false,
       "persistent": true
+    },
+    "clean": {
+      "cache": false
+    },
+    "//#clean": {
```

Comment:
@pedrocarlo is this a typo or some wild Turborepo thing im not aware of?

Reply:
Yep it's a turbo thing that allows you to run scripts at the root level:
https://turbo.build/repo/docs/crafting-your-repository/configuring-tasks#registering-root-tasks

`//` means the "root package" and `#` is the separator between "package" and "task"